packages/webcert/src/utils/TextWithDynamicLinks.tsx

```diff
@@ -0,0 +1,48 @@
+import { sanitizeText } from '@frontend/common'
+import React from 'react'
+import WCDynamicLink from './WCDynamicLink'
+
+interface Props {
+  text: string
+}
+
+const TextWithDynamicLinks: React.FC<Props> = ({ text }) => {
+  const hasDynamicLink = (text?: string): boolean => {
+    if (!text) {
+      return false
+    }
+    return text.split('<LINK:').length > 1
+  }
```

Comment:
`const hasDynamicLink = text.includes('<LINK:')`

Reply:
Partially resolved, but there is no need for having this as a function. text is never undefined so it could simply be changed to `const hasDynamicLink = text.includes('<LINK:')`